src/common/cscs_api_common.py

```diff
@@ -381,7 +392,7 @@ def exec_remote_command(auth_header, system_name, system_addr, action, file_tran
         os.remove(priv_key)
         os.rmdir(temp_dir)
 
-    logging.info(f"Result returned: {result['msg']}")
+    logging.info(f"Result: status_code {result['error']} -> {result['msg']}")
```

Comment:
May be safer to just log a `.format(result)`. Result may not have an `error` key (missing `else` above).

Also, logging at least the `*_errno` variables may also prove helpful in debugging.